### PR TITLE
Issue933: fix Wunused-imports and  Wtype-defaults stack build errors

### PR DESCRIPTION
**Pull request by frankjohnlee**
Request to pull to fix Wunused-imports and  Wtype-defaults stack build errors which appear upon running Stack Build.

### CODE REVIEW

File: app/WebParsing/PostParser.hs [hunk 1]
```diff
@@ -9,14 +9,17 @@ import Data.List
...
 
 fasCalendarURL :: String
 fasCalendarURL = "http://calendar.artsci.utoronto.ca/"
 
+failedString :: String
+failedString = "Failed."
+
 getPost :: String -> IO ()
-getPost str = do
+    getPost str = do
```
Comment (david-yz-liu) [inline app/WebParsing/PostParser.hs]: This shouldn't be indented.

File: app/Svg/Generator.hs [hunk 2]
```diff
@@ -44,6 +44,10 @@ import qualified Data.Map.Strict as M
 import Data.Monoid (mempty, mappend, mconcat)
 import Config (databasePath)
 
+hybridFontSize :: Integer
```
Comment (david-yz-liu) [inline app/Svg/Generator.hs]: Don't change this here; change it in `Css.Constants` if you need to.

Comment (frankjohnlee) [inline app/Svg/Generator.hs]: Okay I will take a look at this thanks David.

File: app/Css/Constants.hs [hunk 3]
```diff
@@ -455,13 +455,14 @@ nodeFontSize :: Num a => a
 nodeFontSize = 12
 
 -- |Defines hybrid font size, 7 in pixels.
-hybridFontSize :: Num a => a
+hybridFontSize :: Integer a => a
```
Comment (david-yz-liu) [inline app/Css/Constants.hs]: This isn't correct - you're mixing up a type class (`Num`) with a concrete type (`Integer`). Use `hybridFontSize :: Integer`. Also, this change wouldn't have compiled, so this is indicating that you aren't compiling your code before submitting a pull request, which you should make sure to always do.

Comment (frankjohnlee) [inline app/Css/Constants.hs]: When I write in the following code:
hybridFontSize :: Num a => a
hybridFontSize = 7
hybridFontSize :: Integer

I get the error:
C:\Users\Frank2\Desktop\Courseography\courseography\app\Css\Constants.hs:460:1: error:
    Duplicate type signatures for `hybridFontSize'
    at app\Css\Constants.hs:458:1-14
       app\Css\Constants.hs:460:1-14

Am I doing this incorrectly David?

**Pull request merged**